### ISSUE TITLE
Pathology foundation model baseline

### ISSUE THREAD

**Issue by susuhu** [Graph-and-Geometric-Learning/STPath#5]
I wonder how you aggregate the pathology foundation model baselines like UNI and Gigapath in the paper?  Cuz when I use a 3-layer MLP after UNI, I got superior results on hest bench on top50 HVG pcc.

**Comment by huangtinglin** [comment]
Thanks for your interest! I used a two-layer MLP with a non-linear activation and layer normalization as the prediction head. For HEST-Bench, we reported results based on all samples rather than using the leave-one-out setting. Is that the approach you applied?

**Comment by susuhu** [comment]
Thanks for the reply! I used three-layer MLP with relu and layernorm with UNI_V1. Trained and validated only on HEST1k and tested on holdout HEST1k bench eval split 0 all samples. I took some joint gene across cancer types which results in a gene panel of size around 3000 cuz I don't have well curated gene panel.  I did STPath inference on the same test samples. PCC is calculated only on top50 HVG. MLP_uni has slightly higher PCC 0.3833 compared to STPath PCC  0.3688.

**Comment by huangtinglin** [comment]
I see. I recommend training and evaluating the model on sample-specific gene panels, since using a shared panel across all cancer types may introduce many low-expression genes (3,000 compared with >20,000 genes per sample on average), according to the fact that >80% of these genes are non-activated (expression level = 0). Training a model on this shared panel could yield better performance by either overfitting to this sparsity pattern or benefiting from a smaller target set (as opposed to the >28,000-gene vocabulary in STPath).

It’s great to see such an exploration! We believe STPath can inspire studies that predict the expression of large-scale gene sets directly through pretraining.

**Comment by susuhu** [comment]
Thanks! I'll try STPath gene vocabulary :)

**Comment by wujialu** [comment]
Hi authors,

Thank you for your great work on STPath.

I have the same question regarding the implementation details of the baseline models (UNI and GigaPath) for the gene expression prediction task. 

I would like to confirm: when you implemented the UNI and GigaPath baselines, did you set the dimension of the prediction head to 38,984 genes, the same as STPath?

Thank you for your clarification!
